simnode/generator.py

```diff
@@ -21,7 +21,8 @@ def __init__(self, lanes: List[Road], obstacles: List[Obstacle], participants: L
         self.participants = participants
         self.time_of_day = time_of_day
 
-    @static_vars(line_width=0.15, num_nodes=100, smoothness=0)
+    # TODO Added markings_smoothing for easier adjustment and experimenting, is this ok?
```

Comment:
That's better. Please remove the TODO and then I'm fine. 👍

Reply:
Should i also remove the "FIXME Recognize changing widths" where i added some basic description?